src/components/EarlyReleaseExemplarUnits/EarlyReleaseExemplarUnits.tsx

```diff
@@ -36,96 +38,107 @@ const EarlyReleaseExemplarUnits: FC<EarlyReleaseExemplarUnitsProps> = ({
   subjectIconBackground,
 }) => {
   return (
-    <MaxWidth $ph={16} $background={color}>
-      <Grid $pt={56} $background={color}>
-        <GridArea $order={[2, 1]} $colSpan={[12, 6]}>
-          <Flex $mb={[56, 0]} $flexDirection={"column"}>
-            <Flex>
-              <Heading
-                $mb={16}
-                $font={["heading-4", "heading-4", "heading-3"]}
-                tag={"h2"}
-              >
-                {heading}
+    <Box $background={color}>
+      <MaxWidth $ph={16} $background={color}>
+        <Grid $pt={56} $background={color}>
+          <GridArea $order={[2, 1]} $colSpan={[12, 6]}>
+            <Flex $mb={[56, 0]} $flexDirection={"column"}>
+              <Flex>
+                <Heading
+                  $mb={16}
+                  $font={["heading-4", "heading-4", "heading-3"]}
+                  tag={"h2"}
+                >
+                  {heading}
+                </Heading>
+                <TagPromotional
+                  size={"large"}
+                  $color={"mint"}
+                  $ml={[16, 16, 24]}
+                  $mt={[8, 8, 10]}
+                />
+              </Flex>
+              <Heading $font={"heading-7"} tag={"h3"}>
+                {subHeading}
               </Heading>
```

Comment:
In Figma, this sub-heading has a font weight of 400, but in the FE it has a font weight of 600:
![Screenshot 2023-09-07 at 10 33 31](https://github.com/oaknational/Oak-Web-Application/assets/3018395/050a282a-4256-4096-9878-e000df96993a)

Reply:
I can see that the Heading 7 styles have been applied as per Figma though, so unless it's possible to override this aspect in this particular it may not be practical to fix this.

Reply:
Ah, in Figma it's `Headings/Light/Heading 7` for the main sub-heading versus `Headings/Regular/Heading 7` for the adjacent sub-heading (`— Sophie Baker`)

Reply:
Good spot, will change this one.